src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -136,6 +136,12 @@ internal static partial class CSharpCodeStyleOptions
             "csharp_style_prefer_range_operator",
             "TextEditor.CSharp.Specific.PreferRangeOperator");
 
+        public static readonly Option2<CodeStyleOption2<bool>> PreferUTF8StringLiteral = CreateOption(
+            CSharpCodeStyleOptionGroups.ExpressionLevelPreferences, nameof(PreferUTF8StringLiteral),
+            defaultValue: s_trueWithSuggestionEnforcement,
+            "csharp_style_prefer_utf8_string_literal",
+            $"TextEditor.CSharp.Specific.{nameof(PreferUTF8StringLiteral)}");
```

Comment:
Roaming storage might not be necessary if this is not exposed in Visual Studio Options UI.

If this will be exposed, then AutomationObject needs to be updated for import/export options to persist the option correctly.